feat: add link to semantic PRs app (#78)
ublue-os/website: docs/making-your-own.md

1. Fork the [ublue-os/base](https://github.com/ublue-os/main) repo:
1. Ensure your [GitHub Actions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository) and [GitHub Packages](https://docs.github.com/en/packages) are set up and enabled

> 2. [Optional] Install the [Semantic PRs](https://github.com/marketplace/semantic-prs) GitHub app if you want nice changelogs
> 3. Change the [image name in the action](https://github.com/ublue-os/base/blob/aab8078cfdc7d2354e057a0ca4771d3a53d2df4c/.github/workflows/build.yml#L14) to match what you want to call your image

Reviewer: ```suggestion
1. [Optional] Install the [Semantic PRs](https://github.com/marketplace/semantic-prs) GitHub app if you want nice changelogs
1. Change the [image name in the action](https://github.com/ublue-os/base/blob/aab8078cfdc7d2354e057a0ca4771d3a53d2df4c/.github/workflows/build.yml#L14) to match what you want to call your image
```

   - Changing it to `IMAGE_NAME: beagles` will name the final image: `ghcr.io/yourusername/beagles` - so you'll likely want that to be your cool name instead of `base`

> 4. Choose your own Adventure:

Reviewer: ```suggestion
1. Choose your own Adventure:
```

   - Edit [packages.json](https://github.com/ublue-os/main/blob/main/packages.json) 
     - Add the rpm's you'd like to be included in your image
     - Flatpaks are a work in progress but you'll be able to edit a file

> 5. Generate a keypair

Reviewer: ```suggestion
1. Generate a keypair
```
